plugins/registry.go

```diff
@@ -110,10 +111,17 @@ func LoadRSPlugin(router *mux.Router, p RSPlugin, mw []middleware.Middleware) er
 // that plugin.
 func loadRoutes(router *mux.Router, p nameRoutes) error {
 	for _, r := range p.Routes() {
+		// If matcher is not defined, define a dummy one that returns
+		// true
+		if r.Matcher == nil {
```

Comment:
@deepjyoti30 A router can define `matcherFunc` if there is a need. What is the requirement on default value? Would it break if a route doesn't have matcher function.

Reply:
Yes, if we define the matcher for some routes and not all, it breaks. That’s why a dummy one is defined that is essentially equivalent of not being present.